main.go

```diff
@@ -17,14 +17,17 @@ import (
 
 	"regexp"
 
+	"crypto/tls"
+	"net/http"
+
 	"golang.org/x/crypto/ssh/terminal"
 
 	log "github.com/sirupsen/logrus"
 	"github.com/docker/distribution/context"
 	schema2 "github.com/docker/distribution/manifest/schema2"
 	"github.com/docker/distribution/reference"
 	"github.com/docker/distribution/registry/client"
-	"github.com/fraunhoferfokus/deckschrubber/util"
+	"github.com/heroku/docker-registry-client/registry"
```

Comment:
This causes me a bit of unease as I always prefer to keep the dependencies as small as possible (see for example how @jgimenez solved this problem [here](https://github.com/fraunhoferfokus/deckschrubber/pull/33/commits/d77f95e70a27535fae511847b249bf0246f7f71f)), but I suppose heroku is trustworthy enough.